### ISSUE TITLE
What is the cls label you used for ReferIt3D fine-tuning?

### ISSUE THREAD

**Issue by RunsenXu** [3d-vista/3D-VisTA#5]
Dear 3D-VisTA authors,

Thanks for your interesting work. 
I wonder what class labels you used for fine-tuning ReferIt3D.

Best,
Runsen

**Comment by zhuziyu-edward** [comment]
We use scannet raw 607 classes for all settings.

**Comment by RunsenXu** [comment]
Dear Ziyu,

But in ReferIt3D's setting, only ground truth bounding boxes are provided and no class labels should be provided. Am I correct?

Best
Runsen

**Comment by zhuziyu-edward** [comment]
Yes, during the Referit3D setting, only ground truth bounding boxes are provided for validation and no class labels are provided. But ground truth label is used for training.

**Comment by RunsenXu** [comment]
I see, thanks for answering.
